frontend/src/metabase/lib/urls/dataApps.ts

```diff
@@ -0,0 +1,13 @@
+import slugg from "slugg";
+
+import { DataApp } from "metabase-types/api";
+
+import { appendSlug } from "./utils";
+
+function dataAppInternal(app: DataApp) {
```

Comment:
could this function name maybe be a little more descriptive?
```suggestion
function dataAppInternalPath(app: DataApp) {
```

Reply:
Fixed! Sorry I committed stuff myself to push once not to disturb the CI too much